### PR TITLE
[merged] commctl can now fall back to other instances.

### PR DESCRIPTION
**Pull request by ashcrow**
When a connection to a commissaire server fails commctl can now attempt to use another instance. Instances are defined via the endpoint parameter in the .commissaire.json configuration file.

### CODE REVIEW

File: src/commctl/client_script.py [hunk 1]
```diff
@@ -298,6 +364,11 @@ def main():
         print(yaml.dump(
             call_result, default_flow_style=False,
             Dumper=yaml.SafeDumper, explicit_end=False).strip())
+    except NoMoreServersError as nmse:
+        print("No servers could be reached. Tried: {0}.".format(
+            ", ".join(nmse.args)))
```
Comment (mbarnes) [inline src/commctl/client_script.py]: Nit: it would be more readable to list each endpoint on a separate line.

**Comment by mbarnes** [comment]
Looks good, just had one little nit.

File: src/commctl/client_script.py (hunk 1 above)
Comment (ashcrow) [inline src/commctl/client_script.py]: That's a good idea. I'll update.

**Comment by mbarnes** [comment]
@cgwalters-bot r+

**Pull request closed without merging**